### PR TITLE
Update properties for entities on recurring costs

### PR DESCRIPTION
**Pull request by drewsshughes**

### CODE REVIEW

**Comment by drewsshughes** [comment]
@tylerwoonton

**Comment by tylerwoonton** [comment]
Just need to fix the tests and this'll be good to go.

File: src/Billing/Entities/RecurringCost.php [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 /**
  * @property int $id
- * @property string $name
+ * @property Type $type
```
Comment (tylerwoonton) [inline src/Billing/Entities/RecurringCost.php]: We don't seem to have an Entity for `Type`. Can we add one please, @drewsshughes?

In the same way that you've done the `Product` Entity.

Comment (tylerwoonton) [reply src/Billing/Entities/RecurringCost.php]: As a result of adding this, we need to make sure the class is imported at the top of the file.

**Comment by tylerwoonton** [comment]
We don't need these changes anymore as a result of #252 

Can you close this PR please @drewsshughes? 👍

**Pull request closed without merging**